ingestors/kafka/src/main/scala/hydra/kafka/programs/CreateTopicProgram.scala

```diff
@@ -124,7 +123,8 @@ final class CreateTopicProgram[F[_]: Bracket[*[_], Throwable]: Sleep: Logger](
 
   def createTopicFromMetadataOnly(topicName: Subject, createTopicRequest: TopicMetadataV2Request): F[Unit] =
     for {
-      _ <- keyAndValueSchemaV2Validator.validate(createTopicRequest, topicName)
+      validator <- eff.delay(KeyAndValueSchemaV2Validator.make(schemaRegistry))
```

Comment:
Do we really need create new instance of `KeyAndValueSchemaV2Validator` for each request?

Reply:
Validator should be created **once, inside**, when TopicCreationProgram is created. It will be right. But for this approach we need to rework a lot of code in both projects, because creating of TopicCreationProgram will return F[TopicCreationProgram[F]].
I left it for the future, when we get rid of IO and rewrite creating services in a functional way - because creating a new object - it’s a side effect. 
The purpose of this move - is to make fewer changes in kstreams.

Reply:
Will rewrite it factory method